docs/docs/concepts/promotion.rst

```diff
@@ -0,0 +1,72 @@
+Promotion
+=========
+
+
+Klaw allows users to control the creation of resources in their Kafka environment through a concept of promotion.
+
+- This adds an extra layer of security avoiding risk of manually duplicating entries across environments.
+- The requests are properly reviewed and verified by another pair of eyes, maintaining sanity of the application.
+- Information is audited. Who is raising the request, when it has been created, and who approved and when. Helps in identifying problems during unexpected behaviour of system.
+- By maintaining a history of applied changes, easy track back on the evolution of a configuration.
+
+
+Promotion of resources are administered by the resource owners, each team has the ability to promote a topic, schema or connector from one environment to the next.
+
+Configure environment promotion
+-------------------------------
+    Server Configuration
+      Found under **Dashboard -> Settings** the Superadmin user has the ability to define the ``Tenant Model`` which configures which environments promote to the next environment in your organisation.
+      Each resource Topic/Schema must already be created by the superadmin under ``Environments`` before being added to the ``Tenant Model`` or the server will reject the configuration.
+      Below in the example we can see that the Kafka Topic environments have been defined as 'Dev' & 'TST' the order is also specified which is the order that will be enforced in the promotion of topics.
+      Similarily the Schema Registry environments have all been defined and placed in order.
+```
+{
+  "tenantModel":
+    {
+      "tenantName": "default",
+      "baseSyncEnvironment": "DEV",
+      "orderOfTopicPromotionEnvsList": ["DEV", "TST"],
+      "requestTopicsEnvironmentsList": ["DEV", "TST"],
+      "orderOfSchemaPromotionEnvsList" : [ "DEV_SCH", "TST_SCH" ],
+      "requestSchemaEnvironmentsList" : [ "DEV_SCH", "TST_SCH" ]
+    }
+}
+
+```
+
+
+
+    Topic Promotion
+      Once a topic is created in the base environment it is then possible to promote this to the next environment.
+      This will create a promotion request that can be reviewed and approved or declined by the requesters team mates. Each created environment is able to be seen in the same ``Topic Overview`` where the topic can be promoted.
```

Comment:
```suggestion
      This will create a promotion request that can be reviewed, approved, or declined by the requester's teammates. Each created environment can be seen in the same ``Topic Overview`` where the topic can be promoted.
```